src/components/table/LightTable.jsx

```diff
@@ -30,6 +30,20 @@ const styles = (theme) => ({
   row: {
     borderRadius: `${theme.shape.borderRadius}px ${theme.shape.borderRadius}px 0 0`,
   },
+  evenRow: {
+    backgroundColor: 'white',
+  },
+  oddRow: {
+    backgroundColor : 'rgba(233,236,239,0.4)'
```

Comment:
just to be consistent with other places in the code, i would probably have these as hex values and then set the `opacity` separately